backend/db/utils.py

```diff
@@ -123,7 +123,10 @@ def get_dependent_tables_queue(independent_tables: List[str]) -> List[str]:
 
 
 def refresh_any_dependent_tables(con: Connection, independent_tables: List[str] = CORE_CSET_TABLES, schema=SCHEMA):
-    """Refresh all derived tables that depend on independent_tables"""
+    """Refresh all derived tables that depend on independent_tables
+
+    :param independent_tables: Any tables that changed for which we now want to update any dependent tables.
```

Comment:
## Minor: Added some documentation for meaning of `independent_tables` param in some refresh functions.